libs/components/layout/src/lib/modules/fluid-grid/scss/_variables.scss

```diff
@@ -1,15 +0,0 @@
-@use 'libs/components/theme/src/lib/styles/variables' as *;
-
-$gutter-sizes: (
```

Comment:
Since these gutter values are now overrides and will go away shortly anyway - I just went ahead and moved them into the files that use them as overrides (which made the selector for those overrides easier) and then moved the `$sky-fluid-grid-columns: 12 !default;` variable into the file that used it as it was the lone remaining one.